cypress/e2e/finance/fiscalYears/rollover-fiscal-year.cy.js

```diff
@@ -17,7 +17,7 @@ import NewOrganization from '../../../support/fragments/organizations/newOrganiz
 import NewInvoice from '../../../support/fragments/invoices/newInvoice';
 import Invoices from '../../../support/fragments/invoices/invoices';
 
-describe('ui-finance: Funds', () => {
+describe('ui-finance: Fiscal Year', () => {
```

Comment:
* Please uncomment or delete after block which starts on line 146 
* line 8: 'Users' is defined but never used 
* line 56: 'user' is assigned a value but never used.

so, if you don't need them, please remove

Reply:
periodStart and periodEnd are duplicated

Reply:
could you please add a comment why do you need the waiter on 118 line